app/controllers/events/meetsReqsForEvent.py

```diff
@@ -0,0 +1,17 @@
+from app.models.user import User
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.programBan import ProgramBan
+from app.models.preqForProgram import PreqForProgram
+
+def isEligibleForProgram(event, user):

```

Comment:
Adding some docstring comments here detailing what this function does is a good idea.